.gitignore

```diff
@@ -69,6 +69,7 @@ include/pulsar/Version.h
 *.ninja*
 .clangd/
 compile_commands.json
+.cmake-build-debug/
```

Comment:
Please revert this unrelated change. The cmake build directory can be any directory.

Reply:
Anyways. I have removed this now.

Reply:
note: cmake-build-debug is a default directory that Jetbrains Clion IDE creates when the project is compiled.

Reply:
> Is this really worth worrying about?

Not a serious problem. But it's better to explain why did you add a new item into the `.gitignore`.

> note: cmake-build-debug is a default directory that the Jetbrains Clion IDE creates when the project is compiled.

And I agree that this explanation is reasonable.